README.md

```diff
@@ -4,10 +4,18 @@
 [![travis-img]][travis-link]
 
 TagBot creates tags and releases for your Julia packages when they're registered.
```

Comment:
```suggestion
TagBot creates tags and releases for your Julia packages when they're registered, so that your tags/releases on GitHub are kept in sync with releases you have made on the Julia package registry.
```